regression/pages/ecommerce/back_to_basket_page.py

```diff
@@ -0,0 +1,53 @@
+"""
+Pages where user is landed after clicking back button on checkout page
+"""
+from bok_choy.page_object import PageObject
+
+from regression.pages.whitelabel.const import ECOMMERCE_URL_WITH_AUTH
+
+
+class BackToBasketPage(PageObject):
+    """
+    Error page when user uses back button from payment page
+    """
+
+    url = ECOMMERCE_URL_WITH_AUTH + 'basket'
+
+    def is_browser_on_page(self):
+        """
+        Is browser on the page?
+        Returns:
+                True if error message container is present on the page:
+        """
+        return self.q(css='.depth.depth-2.message-error').present
+
+    @property
+    def error_message_header(self):
+        """
+        Get error message header
+        Returns:
```

Comment:
You can remove this line and the next one. You should also rename the method to `get_error_message_header`.

A better approach would be to have this method return the error message and then--wherever you're using this, grab the header from it at that point.